content/docs/apps/managed-services.md

```diff
@@ -23,23 +23,6 @@ In order to create a service instance and binding for use with an application, w
 % cf marketplace
 ```
 
-**Output:**
-
-```
-Getting services from marketplace in org ORG / space SPACE as USERNAME
-OK
-
-service                     plans                                    description   
-elasticsearch-swarm-1.7.1   3x, 6x, 1x                               Elasticsearch 1.7.1   
-rds                         shared-psql, micro-psql*, medium-psql*   RDS Database Broker   
-redis28-swarm               standard                                 Redis 2.8 service for application development and testing   
-s3                          basic*                                   Amazon S3 is storage for the Internet.   
-
-* These service plans have an associated cost. Creating a service instance will incur this cost.
-
-TIP:  Use 'cf marketplace -s SERVICE' to view descriptions of individual plans of a given service.
-```
-
```

Comment:
I figure this information is going to go out of date quickly, so wanted to get it out of the docs.